src/simplay-jupyter/src/widget.ts

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) team-simplay
+// Distributed under the terms of the Modified BSD License.
+
+import {
+  DOMWidgetModel,
+  DOMWidgetView,
+  ISerializers,
+} from '@jupyter-widgets/base';
```

Comment:
Looking at the single quote, can we introduce prettier for this typescript project and apply the same rules as for simplay-web?
If thats to much effort create a separate task.